Pipeline/CreateUCDComponentVersion/dbb-ucd-packaging.groovy

```diff
@@ -329,13 +360,13 @@ xml.manifest(type:"MANIFEST_SHIPLIST"){
 			deletedFiles.each { deletedOutput ->
```

Comment:
@M-DLB 

We need to clean the `deleteOutput` as well, like you do when you put it into the HashMap.
```
String cleansedDeletedFile = ((String) deletedFile).replace('"', '');
```